packages/admin-ui/src/page/page-header.tsx

```diff
@@ -40,71 +43,65 @@ import { Box } from '../box'
  *  </PageHeaderBottom>
  * </PageHeader>
  */
-export const PageHeader = createComponent<'header', PageHeaderOptions>(
-  (props) => {
-    const { onPopNavigation, children, ref, ...htmlProps } = props
-    const [scrollOnTop, setScrollOnTop] = useState(true)
-    const [distanceFromTop, setDistanceFromTop] = useState(0)
-    const pageHeaderViewportFakeRef = useRef<HTMLDivElement>(null)
-    const pageHeaderRef = (ref as any) ?? useRef<'header'>(null)
-    const { Portal } = usePortal()
+export const PageHeader = forwardRef(function PageHeader(
+  props: PageHeaderProps,
+  ref: Ref<HTMLElement>
+) {
+  const { className = '', onPopNavigation, children, ...htmlProps } = props
 
-    useEffect(() => {
-      pageHeaderRef?.current?.offsetTop &&
-        setDistanceFromTop(pageHeaderRef.current.offsetTop)
-    }, [])
+  const [scrollOnTop, setScrollOnTop] = useState(true)
+  const [distanceFromTop, setDistanceFromTop] = useState(0)
+  const pageHeaderViewportFakeRef = useRef<HTMLDivElement>(null)
+  const pageHeaderRef = (ref as any) ?? useRef<'header'>(null)
```

Comment:
Hooks should not be called conditionally. You can use the hook `useForkRef` from our utils

Reply:
Done!